src/blueprints/tv2/action-factories/tv2-split-screen-action-factory.ts

```diff
@@ -231,6 +231,26 @@ export class Tv2SplitScreenActionFactory extends ActionFactory {
     }
   }
 
+  private createSplitScreenPieceInterfaceFromPiece(piece: Piece, metadata: Tv2PieceMetadata, timelineObjects: Tv2BlueprintTimelineObject[]): Tv2PieceInterface {
+    return {
+      id: `${piece.getPartId()}_piece`,
```

Comment:
Won't this give multiple pieces with the same id if I e.g. change both input 1 and 2 in a split screen?

Reply:
It would. Updated to also include the timestamp of `Date.now()` to ensure uniqueness.